script.js

```diff
@@ -1,4 +1,400 @@
+    else if (dealerCardSum <= 21) {
+      let output = `The sum of the dealer's cards are: ${dealerCardSum}.<br>`;
+      for (const [i, { playerName, sumOfCards }] of playerProfiles.entries()) {
+        if (dealerCardSum === sumOfCards) {
+          output += `${playerName} has the same cards as the dealer.<br>`;
+          playerProfiles[i].draw += 1;
+        } else if (dealerCardSum > sumOfCards) {
+          output += `${playerName} lost to the dealer with ${sumOfCards}.<br>`;
+          playerProfiles[i].loss += 1;
+        } else if (sumOfCards > 21) {
+          output += `${playerName} lost by going over 21.<br>`;
+          playerProfiles[i].loss += 1;
+        } else if (dealerCardSum < sumOfCards && sumOfCards <= 21) {
+          output += `${playerName} won the dealer with ${sumOfCards}.<br>`;
+          playerProfiles[i].win += 1;
+        }
+      }
+      gameMode = 'start';
+      return output;
+    }
+  }
+  roundsPlayed += 1;
+  scoreboardEl.textContent = outputScoreboardData();
+  return `Starting next round!`;
 };
+////////
+/// MAIN FUNCTION
+/////////////////
+
+// score instead of betting
+const outputScoreboardData = function () {
+  let output = '';
+  for (let [i, { playerName, win, loss, draw }] of playerProfiles.entries()) {
+    output += `${playerName} -> win: ${win}, draw: ${draw}, loss: ${loss}.\r\n`;
+  }
+  return output;
+};
+
+// reset cards in players' and dealer's hands
+const resetCards = function () {
+  dealerCards = [];
+  for (let i = 0; i < playerProfiles.length; i++) {
+    playerProfiles[i].hand = [];
+  }
+};
+
+// check for blackJack
+// black jack = ace + 10/picture cards (max 2 cards)
+const blackJackCheck = function (hand) {
+  if (hand.length === 2 && sumOfCards(hand) === 21) {
+    return true;
+  } else {
+    return false;
+  }
+};
+
+const sumOfEachPlayerCards = function () {
+  // loop through each player's hand, add new object sum for total
+  for (const [i, { hand }] of playerProfiles.entries()) {
+    let sum = sumOfCards(hand);
+    playerProfiles[i].sumOfCards = sum;
+  }
+};
+
+// what does the active player do?
+// TODO refactor code in !input and input
+const hitOrStand = function (input) {
+  // show cards for the active player
+  inputEl.placeholder = `Type 'Hit' or 'Stand' and submit.`;
+  if (!input) {
+    // give option to hit or stand
+    let hand = playerProfiles[activePlayer].hand;
+    let sumOfCurrCards = sumOfCards(hand);
+    let outputStr = `Player ${
+      activePlayer + 1
+    } has these cards in hand, please enter 'Hit' or 'Stand':`;
+    for (let i = 0; i < hand.length; i++) {
+      outputStr += `<br> ${hand[i].name} of ${hand[i].suit}.`;
+    }
+    outputStr += `<br><br>Sum of cards: ${sumOfCurrCards}`;
+    // add score into player object.
+    playerProfiles[activePlayer].cardSum = sumOfCurrCards;
+    return outputStr;
+  }
+  // if input is true, check for hit / stand
+  input = input.toLowerCase();
+  if (typeof input === 'string') {
```

Comment:
Why do you need to check for typeof input? Doesn't checking for 'hit' or 'stand' already check for whether or not it's a string?